libraries/ESP32/examples/Zigbee/Zigbee_Temperature_Sensor/README.md

```diff
@@ -0,0 +1,83 @@
+# Arduino-ESP32 Zigbee Temperature Sensor Example
+
+This example shows how to configure the Zigbee end device and use it as a HA temperature sensor.
```

Comment:
I think we can change "HA" to "Home Automation (HA)" for the first mention only.